web/styles/app_variables.css

```diff
@@ -581,6 +581,25 @@
     */
     --table-header-sortable-column-padding-right: 1em;
 
+    /* Banner grid layout variables */
+    --banner-horizontal-padding: 13px;
+    --banner-vertical-padding: 5px;
+    --banner-grid-template-areas-lg: ". . . . . . banner-close-btn banner-close-btn"
+        ". . banner-label . banner-btns . banner-close-btn banner-close-btn"
+        ". . . . . . banner-close-btn banner-close-btn";
+    --banner-grid-template-columns-lg: var(--banner-horizontal-padding) 0 auto
+        minmax(0, 1fr) auto 0 minmax(0, auto) var(--banner-horizontal-padding);
+    --banner-grid-template-rows-lg: 5px auto 5px;
+    --banner-grid-template-areas-md: ". . . . banner-close-btn banner-close-btn"
+        ". . banner-label banner-label banner-close-btn banner-close-btn"
+        ". banner-btns banner-btns banner-btns . .";
```

Comment:
`s/btn/button/` in these grid label areas seems like it'd be nice? @karlstolley thoughts?

Reply:
Yes. Nothing is gained by `btn`

Reply:
Sorry, I missed this. Renamed btn -> button.